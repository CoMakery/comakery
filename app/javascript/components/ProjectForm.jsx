import React from 'react'
import PropTypes from 'prop-types'

class ProjectForm extends React.Component {
  constructor(props) {
    super(props)

    this.state = {
    }
  }

  render() {
    return (
      <React.Fragment />
    )
  }
}

ProjectForm.propTypes = {
}
ProjectForm.defaultProps = {
}
export default ProjectForm
