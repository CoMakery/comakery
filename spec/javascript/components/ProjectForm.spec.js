import React from 'react'
import { mount } from 'enzyme'
import ProjectForm from 'components/ProjectForm'

describe('ProjectForm', () => {
  it('renders correctly without props', () => {
    const wrapper = mount(<ProjectForm />)

    expect(wrapper).toMatchSnapshot()
  })
})
