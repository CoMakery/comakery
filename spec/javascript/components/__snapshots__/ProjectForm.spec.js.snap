// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectForm renders correctly without props 1`] = `<ProjectForm />`;
